packages/core/src/components/tree/tree.tsx

```diff
@@ -63,10 +63,6 @@ export class Tree<T = {}> extends React.Component<ITreeProps<T>, {}> {
 
     private nodeRefs: { [nodeId: string]: HTMLElement } = {};
 
-    constructor(props?: ITreeProps<T>, context?: any) {
```

Comment:
@izikorgad optional `props` is not compatible with [React typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L281-L282) in strict mode, so we simply have to define the `props` argument in all the constructor types.

Reply:
That's right...